vercel_blob/blob_store.py

```
@@ -128,12 +162,13 @@ def list(options: dict = {}) -> dict:
         'GET',
         params=params,
         headers=headers,
+        timeout=timeout,
     )
 
     return _response_handler(resp)
 
 
-def put(path: str, data: bytes, options: dict = {}) -> dict:
+def put(path: str, data: bytes, options: dict = {}, timeout: int = 10, verbose: bool = False) -> dict:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)